pallets/asset-registry/Cargo.toml

```diff
@@ -19,6 +19,9 @@ codec = { default-features = false, features = ["derive", "max-encoded-len"], pa
 primitive-types = { default-features = false, version = "0.8.0" }
 serde = { features = ["derive"], optional = true, version = "1.0.101" }
 
+# HydraDX dependencies
+hydra-dx-traits = { git = "https://github.com/galacticcouncil/hydradx-traits", default-features = false }
+
 # Local dependencies
 primitives = { path = "../../primitives", default-features = false }
```

Comment:
primitives - I believe this could be now as dev-dependency only as it is used only it tests. Or remove the dependency completely and define the needed types directly in the tests.

This can be the case in other pallets too.